src/modules/popup.js

```diff
@@ -25,4 +39,22 @@ export default async (
     popups.style.display = 'none';
     popupContainer.innerHTML = '';
   });
+
+  const form = document.querySelector('.form');
+  form.addEventListener('submit', (e) => {
+    e.preventDefault(); // prevent the default from submission.
+    const name = document.getElementById('name').value;
+    const insights = document.getElementById('insights').value;
+    comments(id, name, insights);
+  });
+  const commentsDiv = document.querySelector('.commentsHolder');
+  commentContent.forEach((obj) => {
+    commentsDiv.innerHTML += `
+    <div class="commentaries">
+    <p class="commentName">Name= ${obj.username}</p>
+    <p class="commentCom">Comment= ${obj.comment}</p>
+    <p class="creationDate">Date and Time=${obj.creation_date}</p>
```

Comment:
please consider changing the order and put the date first in order of improve the order of the comments